AIRO-356: Tutorial Updates

at669 opened the pull request:
- More explanation on location of windows, menu options
- Highlighted sections of screenshots in Part 1
- Minor clarifications, additional screenshots
- Moved the default target/targetPlacement objects to the "front" (on the Unity logo) such that the action happens "in front of the camera" as seen below:
    ![Screen Shot 2021-01-06 at 1 57 18 PM](https://user-images.githubusercontent.com/31416491/103819163-1dde8100-5027-11eb-8b20-3702712ac9fb.png)
- Removed references to Message Generation window, updated to Robot menu options & Browse window
- Removed instructions to set up RosConnect object, updated scripts & instructions to reference `ROSConnection.instance`
- Updated miscellaneous screenshots to reflect these changes

mpinol commented on tutorials/pick_and_place/2_ros_tcp.md:
```diff
@@ -39,13 +39,13 @@ To enable communication between Unity and ROS, a TCP endpoint running as a ROS n
    ![](img/2_single.png)
 
-   In the Message Auto Generation window click `Browse File...` and navigate to the MoveIt Msgs repository. Select `moveit_msgs/msg/RobotTrajectory.msg`, and then click `GENERATE!` 
+   In the Message Auto Generation window click `Browse File...` and navigate to the MoveIt Msgs directory in this cloned repository (`Unity-Robotics-Hub/tutorials/pick_and_place/ROS/src/moveit_msgs`). Select `moveit_msgs/msg/RobotTrajectory.msg`, and then click `GENERATE!` 
```
Would it make sense to change the message generation steps to use the new MessageGenerationBrowser?

LaurieCheers-unity commented on tutorials/pick_and_place/2_ros_tcp.md:
```diff
@@ -119,23 +126,27 @@ To enable communication between Unity and ROS, a TCP endpoint running as a ROS n
+1. Next, the ROS TCP connection needs to be defined. Select `Robotics -> ROS Settings` from the top menu bar. 
 
-   ![](img/2_target.gif)
+   In the ROS Settings window, the `Host Name` should be the IP address of your ROS machine (*not* the one running Unity).
```
The field is called "ROS IP Address"

sarah-gibson reviewed (approved):
Approved, pending:
1) fix the echo ROS_IP problem in Dockerfile
2) check on the issue with not being able to pick up the cube

at669 commented:
> 1. fix the echo ROS_IP problem in Dockerfile

removed echo from Dockerfile, moved to `set up workspace` script. appears to be functional for me--added some troubleshooting tips just in case

> 2. check on the issue with not being able to pick up the cube

swapped gripper open/close values to match changes in Unity-Technologies/Unity-Robotics-Hub#84
